doc/integrator/objectstorage.rst

```diff
@@ -44,9 +44,10 @@ Create the vrt file for a raster layer:
 
 .. prompt:: bash
 
-   docker-compose exec geoportal ${'\\'}
-        gdalbuildvrt /vsis3/<bucket>/<folder>/index.vrt \
-        $(list4vrt <bucket> <folder>/)
+   docker-compose exec geoportal bash -c ${'\\'}
+        'gdalbuildvrt /vsis3/<bucket>/<folder>/index.vrt \
+        $(list4vrt <bucket> <folder>/ .tif)'
```

Comment:
```suggestion
        $(list4vrt <bucket> <folder>/ ".tif")'
```
Don't know if it makes a difference but ``.tif`` was double-quoted in the command you sent me for Prilly.

Reply:
No differences...